CredentialCardField.vue

```diff
@@ -60,7 +61,17 @@
         <span>{{name}}</span>
       </div>
       <q-item-section
-        v-if="!sublabels"
+        v-if="component === 'Image'"
+        class="g-field-data-regular">
+        <q-img style="max-width: 128px" class="rounded-borders" :src="value" />
+      </q-item-section>
+      <q-item-section
+        v-else-if="component === 'FixedWidth'"
```

Comment:
If you want to haggle over naming, FixedWidthText might be better.  Or maybe PreformattedText.